common/src/main/java/net/whimxiqal/journey/search/PathTrial.java

```diff
@@ -28,8 +28,7 @@
 import net.whimxiqal.journey.Cell;
 import net.whimxiqal.journey.navigation.Mode;
 import net.whimxiqal.journey.navigation.Path;
-import net.whimxiqal.journey.search.function.CostFunction;
-import net.whimxiqal.journey.search.function.PlanarOrientedCostFunction;
+import net.whimxiqal.journey.search.function.*;
```

Comment:
I prefer to use explicit imports rather than star imports, so please import individually

Reply:
Classic IntelliJ moment, will fix.

Reply:
Separated the imports.